packages/evolution-generator/src/scripts/generate_widgets.py

```
@@ -11,7 +11,7 @@
 )
 import re  # Regular expression module for pattern matching
 from typing import TypedDict
-from dataclasses import dataclass
+from dataclasses import dataclass, field
```

Ruff (0.12.2)

`dataclasses.field` imported but unused

Remove unused import: `dataclasses.field`

(F401)